scripts/build.sh

```diff
@@ -1,11 +1,8 @@
 # Usage: `npm run build`
 # NPM establishes a reliable working directory
```

Comment:
I'm curious - what do you mean by that line ?

Reply:
Good question!

If you've tried using relative paths in bash scripts, then you've probably seen discussions like 
https://stackoverflow.com/q/59895 and http://mywiki.wooledge.org/BashFAQ/028.

It's notoriously tricky to get this right. However, `npm run` provides some useful extra guarantees - it always executes the script with the working directory set to the directory that contains `package.json`, regardless of which subdirectory `npm run` was executed from. 

That's why I didn't include a `#! /bin/bash` or similar, and instead recommended `npm run`.

Does that make sense?